src/kemal/param_parser.cr

```diff
@@ -42,7 +42,7 @@ class Kemal::ParamParser
 
     body = @request.body as String
 
-    case json = JSON.parse(body)
+    case json = JSON.parse(body).raw
```

Comment:
because in Crystal 0.10.0
`JSON.parse now returns JSON::Any, which allows traversal of JSON objects with less casts`

then we have to use the raw method in `JSON`class:
http://crystal-lang.org/api/JSON/Any.html#raw-instance-method